templates/conference/cart/step_2_pick_tickets.html

```diff
@@ -51,7 +51,7 @@ <h4>{{ fare.name }}</h4>
                         </td>
                     </tr>
                     {% empty %}
-                    No tickets available
+                    No tickets available - if you're looking for early bird tickets, it means they have sold out.
```

Comment:
I'd reword this to the more generic: "No tickets available, sorry. This could mean that ticket sales in the current category have not yet started or the tickets are already sold out (early bird tickets tend to sell out quickly) ".